src/Form/matching-config/selectors-css.js

```diff
@@ -1,7 +1,12 @@
-const formInputsSelector = `
-input:not([type=submit]):not([type=button]):not([type=checkbox]):not([type=radio]):not([type=hidden]):not([type=file]):not([type=search]):not([type=reset]):not([type=image]):not([name^=fake i]):not([data-description^=dummy i]):not([name*=otp]):not([autocomplete="fake"]),
-[autocomplete=username],
-select`
+// We've seen non-standard types like 'user'. This selector should get them, too
+const genericTextField = `
+input:not([type=button]):not([type=checkbox]):not([type=color]):not([type=file]):not([type=hidden]):not([type=radio]):not([type=range]):not([type=reset]):not([type=image]):not([type=search]):not([type=submit]):not([type=time]):not([type=url]):not([type=week]):not([name^=fake i]):not([data-description^=dummy i]):not([name*=otp]):not([autocomplete="fake"]):not([placeholder^=search i]):not([type=date]):not([type=datetime-local]):not([type=datetime]):not([type=month])`
+
+const formInputsSelector = [
+    genericTextField,
+    '[autocomplete=username]',
+    'select'
+]
```

Comment:
I noticed that there was a lot of duplication and incomplete selectors between these two, so I've moved them closer, and derive one from the other to avoid duplication and one getting out of sync with the other. The meaningful change here is to add `:not([placeholder^=search i])` which skips more search fields.